roles/core/node/tasks/install.yml

```diff
@@ -38,6 +47,11 @@
 #
 # Run chosen installation method to get list of RPMs
 #
+- name: install | gpfs base path
+  set_fact:
+    gpfs_path_url: '/usr/lpp/mmfs/{{ scale_version }}/gpfs_rpms'
+  when: scale_install_localpkg_path is defined or scale_install_remotepkg_path is defined
+
```

Comment:
Why would we not put this variable into a defaults file and set something like this

```
gpfs_path_base: "/usr/lpp/mmfs/{{ scale_version }}"
gpfs_path_rpms: "{{ gpfs_path_base }}/gpfs_rpms"
gpfs_path_debs: "{{ gpfs_path_base }}/gpfs_debs"
```